docs/courses/coursemanagement.md

```diff
@@ -254,6 +254,18 @@ You can now
 ![Book Mapping](/img/bookmapping.png)
 
 ## Publishing
+
+As you update your assignment, a **Publish** button will show to the left of your username in the top menu bar.  
+
+![Quick Publish](/img/assignmentpubish.png)
+
+This can be used as a shortcut to publish dialog. There is also an option to publish in the **Education** menu
+
+**Please note**:
+
+- if you change stack or files via terminal it will NOT appear
```

Comment:
"If you change the stack or files via the terminal, the publish button on the top menu bar will NOT appear"